custom_components/better_thermostat/adapters/mqtt.py

```diff
@@ -62,6 +62,8 @@ async def set_hvac_mode(self, entity_id, hvac_mode):
 
 async def get_current_offset(self, entity_id):
     """Get current offset."""
+    if self.real_trvs[entity_id]["local_temperature_calibration_entity"] is None:
+        return 0
```

Comment:
Sry, but this wasn't the soultion, it have to be in the startup. i have fixed it already so you can remove it

Reply:
https://github.com/KartoffelToby/better_thermostat/blob/c0171177f45c9868151bb703e20e93dd3d001383/custom_components/better_thermostat/climate.py#L609

Reply:
No problem. You're right I just decided to fix it deeper because this problem can back in future.